tools/codegen/gen.py

```diff
@@ -17,6 +17,9 @@
 import tools.codegen.api.dispatcher as dispatcher
 import tools.codegen.api.legacy_dispatcher as legacy_dispatcher
 import tools.codegen.local as local
+from tools.codegen.selective_build.selector import SelectiveBuildBase
+from tools.codegen.selective_build.selector import SelectiveBuildSelectorFactory
+from tools.codegen.selective_build.selector import SelectiveBuildNotSelective
```

Comment:
nit: combine these into one import statement and use backlash if you must. Or do a wildcard import; the general style in the codegen is that wildcards are OK for type imports (and we enforce that all types have unique names)--this matches general Python typing style.